### DISCUSSION TITLE
请问如何配置缩略图？

### DISCUSSION THREAD

**Discussion by imotosaiko** [AlistGo/alist#3779] [Q&A]
在网格视图下不加载缩略图，当我进入一个相册文件夹还好，但是一从相册文件夹出来就需要调整视图，否则不会显示除了图片以外的文件（也就什么都没了）
在讨论里浏览了类似的讨论，似乎缩略图并非alist的功能而是挂载的储存所提供的？但是我不清楚那些储存可以提供这样的缩略图功能，比如我现在正在使用阿里云的对象存储，我找到了其支持对图像处理进行缩放（而一些文章则以此作为缩略图的实现），那么这样的功能可以为alist配置缩略图吗？另外，视频的缩略图又将如何处理呢？
又或者，退而求其次，可以单独为每个文件夹设置视图么？这样子可以仅仅在放图片的地方启用图片视图，尽管不如统一成网格视图更加方便，但也勉强够用。
我浏览了alist文档，没有找到相关的问题答案，故来此询问，希望大神看到后能够不吝赐教，谢谢各位！

**Comment by zhushen12580** [comment]
同问
